techniques/flash/FLASH23.html

```diff
@@ -41,7 +41,7 @@ <h3>Adding a summary to a DataGrid with ActionScript 3</h3>
             <p>This is a basic AS3 example of a DataGrid component that has summary
     						text added as its accessible description. </p>
          
-         <pre xml:space="preserve">import fl.accessibility.DataGridAccImpl;
+         <pre>import fl.accessibility.DataGridAccImpl;
```

Comment:
all the naked `<pre> ... </pre>` blocks in this technique could do with inner `<code> ... </code>`